MeshLib/CoordinateSystem.h

```diff
@@ -47,8 +47,7 @@ class CoordinateSystem
     explicit CoordinateSystem(const Element &ele);
 
     /// Decides a coordinate system from a bounding box
-    template <class T>
-    explicit CoordinateSystem(const GeoLib::AABB<T> &bbox) : _type(getCoordinateSystem(bbox)) {}
+    explicit CoordinateSystem(const GeoLib::AABB &bbox) : _type(getCoordinateSystem(bbox)) {}
```

Comment:
I don't really understand what this whole class does and the documentation is not great. Not the point of this PR though. Is it possible that class has really nothing to do with coordinate systems but with extends and dimensionality of data sets?